code/modules/clothing/glasses/glasses.dm

```diff
@@ -314,3 +314,19 @@
 	icon_state = "kaminaglasses"
 	item_state = "kaminaglasses"
 	cover_hair = 1
+
+/obj/item/clothing/glasses/contacts
+	name = "contact lenses"
+	desc = "Only nerds wear glasses."
+	icon = 'icons/obj/items.dmi'
+	icon_state = "contact"
+	prescription = 1
+	body_parts_covered = ~EYES
```

Comment:
Doesn't this cover literally every body part except the eyes?

Reply:
didn't saw this but yes you need to use &= ~EYES

Reply:
you could do that in the /New() but I am pretty sure &= won't work in a definition because it's an operation, definitions need constant expressions

Reply:
T-thanks BYOND

Reply:
i tried compiling with &= and it failed

Reply:
literally just = EYES if you want to cover the eyes

Reply:
He wants the opposite, he wants it to not cover the eyes (because they're contacts)

In this case though you just want `body_parts_covered = null`

Reply:
thanks i'll commit this in a second